docs-ref-services/latest/identity-readme.md

```diff
@@ -196,7 +194,7 @@ The [Managed identity authentication](https://docs.microsoft.com/azure/active-di
 ####  Authenticating in Azure with Managed Identity
 This examples demonstrates authenticating the `SecretClient` from the [azure-security-keyvault-secrets][secrets_client_library] client library using the `ManagedIdentityCredential` in a virtual machine, app service, function app, cloud shell, or AKS environment on Azure, with system assigned, or user assigned managed identity enabled.
 
-see more about how to configure your Azure resource for managed identity in [Enable managed identity for Azure resources](https://github.com/Azure/azure-sdk-for-java/wiki/Set-up-Your-Environment-for-Authentication#enable-managed-identity-for-azure-resources)
+see more about how to configure your Azure resource for managed identity in [the managed identity documentation](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview).
```

Comment:
remove en-us/

Reply:
Its better just to skip the domain name and start with slash (/azure/active-directory/managed-identities-azure-resources/overview)